aten/src/ATen/native/cuda/DilatedMaxPool2d.cu

```diff
@@ -57,15 +62,91 @@ __global__ void MaxPoolForward(const int nthreads, const scalar_t* bottom_data,
   }
 }
 
-static const int BACKWARD_THREADS = 256;
+template <typename scalar_t, typename accscalar_t>
+C10_LAUNCH_BOUNDS_1(CUDA_MAX_THREADS)
+__global__ void MaxPoolForwardNHWC(const int nthreads, const scalar_t* bottom_data,
```

Comment:
`nthreads` & `num` is not used any more. Same with `MaxPoolBackwardNHWC`

Reply:
Fixed